qiskit_experiments/base_analysis.py

```diff
@@ -90,3 +92,34 @@ def _run_analysis(self, experiment_data, **options):
                    None, a single figure, or a list of figures.
         """
         pass
+
+    @staticmethod
+    def exp_fit_fun(x, a, tau, c):
+        """
+        Exponential fit function
+        """
+        return a * np.exp(-x / tau) + c
+
+    # pylint: disable = invalid-name
+    @staticmethod
+    def curve_fit_wrapper(f, xdata, ydata, sigma, **kwargs):
```

Comment:
We need to be consistent with either type hints or doc string types. These should be something like `f (callable), xdata (ndarray), ydata(ndarray), sigma (ndarray)`

Reply:
done